spec.md

```diff
@@ -997,6 +997,19 @@ message ControllerServiceCapability {
 
 If the plugin is unable to complete the ControllerGetCapabilities call successfully, it MUST return a non-ok gRPC code in the gRPC status.
 
+#### RPC Interactions
+
+##### `CreateVolume`, `DeleteVolume`, `ListVolumes`
+
+It is worth noting that the plugin-generated `volume_id` is a REQUIRED field for the `DeleteVolume` RPC, as opposed to the CO-generated volume `name` that is REQUIRED for the `CreateVolume` RPC: these fields MAY NOT contain the same value.
+If a `CreateVolume` operation times out, leaving the CO without an ID with which to reference a volume, and the CO *also* decides that it no longer needs/wants the volume in question then the CO MAY choose one of the following paths:
+
+1. Replay the `CreateVolume` RPC that timed out; upon success execute `DeleteVolume` using the known volume ID (from the response to `CreateVolume`).
+2. Execute the `ListVolumes` RPC to possibly obtain a volume ID that may be used to execute a `DeleteVolume` RPC; upon success execute `DeleteVolume`.
+3. Do not execute any additional RPCs and instead log an event/message in order to facilitate garbage collection of unused resources at some later point in time.
```

Comment:
Maybe remove this? I think the garbage collector itself will either use 1 or 2 above to cleanup the unneeded volume?

Reply:
Option 3 could be leak and expect user to cleanup.